### PR TITLE
APP-486: update gpio

### PR DESCRIPTION
**Pull request by antonshkurenko**

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

File: gpiocontrol/source/android/app/src/main/java/org/kaaproject/kaa/examples/gpiocontol/activity/GPIOStatusListActivity.java [hunk 1]
```diff
@@ -1,66 +1,96 @@
...
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```
Comment (rasendubi) [inline gpiocontrol/source/android/app/src/main/java/org/kaaproject/kaa/examples/gpiocontol/activity/GPIOStatusListActivity.java:14]: Don't modify license header. (Basically, it should not be javadoc comment.)

**Comment by Slash32** [comment]
@kaa-leeroy add to whitelist

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Comment by Slash32** [comment]
The original commit was found to be already merged into master. The rest of the PR has been re-based as #594 . As of this, closing the PR.

**Pull request closed without merging**